### DISCUSSION TITLE
Can FluxCD move a repository to another repository?

### DISCUSSION THREAD

**Discussion by tanawatcha** [fluxcd/flux2#4475] [Q&A] [answered]
Can FluxCD move a repository to another repository? 🙏

**Comment by hiddeco** [comment, marked as answer]
Flux does not manage the lifecycle of repositories but only performs pull and push operations on them, except for `flux bootstrap` being able to create a new Git repository at a variety of providers to manage a new Flux installation and resources in.

TL;DR: it can not.

**Comment by tanawatcha** [comment]
Thank you 😊
